tests/unit/packager/test_parameters.py

```diff
@@ -21,17 +21,24 @@
 from holoscan_cli.common.enum_types import (
     ApplicationType,
     Platform,
-    PlatformConfiguration,
 )
 from holoscan_cli.common.exceptions import UnknownApplicationTypeError
 from holoscan_cli.packager.parameters import PackageBuildParameters, PlatformParameters
 
 
 class TestPlatformParameters:
-    def test_with_aarch64(self, monkeypatch):
+    @pytest.mark.parametrize(
+        "platform",
+        [
+            (Platform.Jetson),
+            (Platform.IGX_iGPU),
+            (Platform.IGX_dGPU),
+            (Platform.SBSA),
+        ],
+    )
+    def test_jetson(self, platform: Platform):
         build_parameters = PlatformParameters(
-            Platform.IGXOrinDevIt,
-            PlatformConfiguration.iGPU,
+            platform,
```

Comment:
Should this be limited to Jetson or intended for negative testing too?

Reply:
This test case tests 4 Platforms: `Jetson`, `IGX_iGPU`, 'IGX_dGPU', and 'SBSA' using `@pytest.mark.parametrize`.